### DISCUSSION TITLE
朋友们帮帮忙，屏幕不亮可能是什么原因？

### DISCUSSION THREAD

**Discussion by 838162661** [peng-zhihui/ElectronBot#168] [Q&A]
我的主板装上了驱动，刷入程序运行simple.exe显示刷入成功，但是屏幕没有任何显示，我以为屏幕坏了就去测试舵机，等舵机测好换了一块屏幕再刷入程序运行simple.exe就只有连接成功，刷不进去了，测了屏幕电压是3.3V，帮忙分析一下可能是什么原因？USB3300坏了吗

**Comment by peng-zhihui** [comment]
要装驱动的，仔细看README，另外b站有人发教程可以参考一下
